### ISSUE TITLE
Jenkins invalid configuration causes AuthenticationException and UI not respond correctly to this

### ISSUE THREAD

**Issue by krasnyt** [MCMicS/jenkins-control-plugin#227]
I wanted to switch to a different jenkins instance but after changing the connection details in the plugin configuration and testing the connection (successfull), I still see the old builds in the Jenkins view. Clicking refresh doesn't do anything. Reinstalling the plugin kept the configuration and the builds from the old jenkins went away but it doesn't load new ones. 

If Jenkins Url is changed to invalid following happens:
- [x] invalid configuration causes AuthenticationException (Jenkins Url setting: http//localhost:8080/jenkins Idea Plugin Setting; http//localhost:8080/) causes that the error occurs. Improve that check and use it in "Test Connection" Action
- [x] old job list will not refreshed.
- [x] old view are still shown

I see an exception in the event log:
<img width="1388" alt="image" src="https://user-images.githubusercontent.com/6813833/85622075-9327cd00-b666-11ea-88ac-d3b7e1549429.png">

**Comment by MCMicS** [comment]
Strange. Is the jenkins instance fully running? There is a problem if you want to connect to an instance which is currently starting.

Could you please goto settings, click test connection and then click apply

**Comment by krasnyt** [comment]
Unfortunately did not help. Setting wrong username returns "[Fail] Unauthorized -> Missing or bad credentials". Then I correct the username and I get "Successfull" (no exception yet). Then I click Apply and THEN I get the exception above.

**Comment by MCMicS** [comment]
Take a look at this tomorrow

**Comment by krasnyt** [comment]
This jenkins is unfortunately private. I just tried it in different intellij project (only that one opened) configuring it from scratch and the result is the same. We can surely talk.

**Comment by MCMicS** [comment]
In my tests it works fine. Changed the server and the new one is load
Maybe some with the jenkins configuration

> This jenkins is unfortunately private. I just tried it in different intellij project (only that one opened) configuring it from scratch and the result is the same. We can surely discuss it on chat.

Sounds like it is generally an issue in your setup with this jenkins. So the issue title have to be updated

Could you try following curl command (or equivalent):

```curl
curl -v --user name:token http://localhost:8080/api/json

curl -v --user name:token http://localhost:8080/api/json?tree=nodeName,nodeDescription,primaryView%5Bname,url%5D,views%5Bname,url,views%5Bname,url%5D%5D
```
 or use http file in IntelliJ:
```
### Successful test: check response status is 200
GET https://localhost:8443/api/json?tree=nodeName
Authorization: Basic {{username}} {{token}}

> {%
client.test("Request executed successfully", function() {
  client.assert(response.status === 200, "Response status is not 200");
});
%}

###
GET https://localhost:8443/api/json?tree=nodeName,nodeDescription,primaryView%5Bname,url%5D,views%5Bname,url,views%5Bname,url%5D%5D
Authorization: Basic {{username}} {{token}}

> {%
client.test("Request executed successfully", function() {
  client.assert(response.status === 200, "Response status is not 200");
});
%}
###
```
 If there are private data send it in gitter private chat or I can offer u my private mail

In meanwhile I test different jenkins setting

**Comment by MCMicS** [comment]
I also created an version with more log info: 
[jenkins-control-plugin-0.13.6-2020.1-bug227.zip](https://github.com/MCMicS/jenkins-control-plugin/files/4829456/jenkins-control-plugin-0.13.6-2020.1-bug227.zip)

**Comment by krasnyt** [comment]
Let's continue on gitter, I sent you a message.

**Comment by MCMicS** [comment]
released with [0.13.6](../releases/tag/v0.13.6)
